src/components/MaModal/MaModal.css

```
@@ -0,0 +1,131 @@
+.modal-overlay {
+  position: fixed;
+  background-color: var(--shadow-dark);
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+}
+
+.modal {
+  position: fixed;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 96vw; /* so that it doesn't reach end of viewport */
+
+  border-radius: 4px;
+  box-shadow: 0 2px 8px var(--shadow-dark);
+  background-color: var(--color-white-base);
+  word-break: break-word;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: var(--spacing-medium);
+  gap: var(--spacing-medium);
+  border-radius: 4px 4px 0 0;
+}
+
+.modal-header--white {
+  --modal-header-icon-color: var(--color-pink-base);
+}
+
+.modal-header--gradient {
+  --modal-header-icon-color: var(--color-white-base);
+
+  background: linear-gradient(
+    to right,
+    var(--color-orange-base),
+    var(--color-pink-base)
+  );
+  color: var(--color-white-base);
+}
+
+.modal-header--gradient .modal-title {
+  text-align: center;
+}
+
+.modal-title {
+  flex: 1;
+  font-weight: 700;
+  font-size: 1.5rem;
+}
+
+.icon-close {
+  border-style: solid;
+  border-width: 1px;
+  padding: 0.25rem;
+  color: var(--modal-header-icon-color);
+}
+
+.icon-close--pink {
+  color: var(--color-pink-base);
+
+  &:hover,
+  &:active,
+  &:focus {
+    color: var(--color-pink-dark);
+  }
+}
+
+.icon-close--white {
+  color: var(--color-white-base);
+
+  &:hover,
+  &:active,
+  &:focus {
+    opacity: 0.75;
+  }
+}
+
+.modal-content {
+  /* because fuck standards, that's why: https://github.com/w3c/csswg-drafts/issues/129 */
+  margin-bottom: var(--spacing-medium);
+  padding: 0 var(--spacing-medium);
+  overflow-y: auto;
+
+  /* so that it doesn't reach end of viewport */
+  max-height: 85vh;
+}
+
+.modal--width-small {
+  max-width: 300px;
+}
+.modal--width-medium {
+  max-width: 500px;
+}
+.modal--width-large {
+  max-width: 800px;
+}
+
+.modal-transition-leave-to,
+.modal-transition-enter {
+  opacity: 0;
+}
+
+.modal-transition-leave-active,
+.modal-transition-enter-active {
+  transition: opacity 0.3s ease;
+}
+
+.modal-transition-enter-active {
+  .modal {
+    animation: slideFromTop 0.4s ease-in-out;
+  }
+}
```

I believe this is not ok 🤔 

```suggestion
.modal-transition-enter-active .modal {
    animation: slideFromTop 0.4s ease-in-out;
}
```

---

right! fixed 🚀